src/views/TreasuryDashboard/components/Metric/Metric.js

```diff
@@ -78,9 +71,9 @@ export const GOHMPrice = () => {
       className="metric wsoprice"
       label={t`gOHM Price`}
       metric={gOhmPrice && formatCurrency(gOhmPrice, 2)}
-      isLoading={gOhmPrice ? false : true}
+      isLoading={!gOhmPrice}
       {...sharedProps}
-      tooltip={`gOHM = sOHM * index\n\nThe price of gOHM is equal to the price of OHM multiplied by the current index`}
+      tooltip={t`gOHM = sOHM * index\n\nThe price of gOHM is equal to the price of OHM multiplied by the current index`}
```

Comment:
this one messes up the `newline` parsing? 
![image](https://user-images.githubusercontent.com/80423742/149819673-a93fb47c-0b2e-4b0b-81d8-80defabc1ed3.png)

These are just nuances of lingui usage.